### DISCUSSION TITLE
List Received Documents documenti non registrati

### DISCUSSION THREAD

**Discussion by andry80pa** [fattureincloud/api#465] [Q&A] [answered]
Buonasera, è possibile leggere da questa API anche i documenti marcati come "da registrare"? Al momento riesco a vedere solo quelli registrati. Grazie.

**Comment by valmoz** [comment, marked as answer]
Buongiorno @andry80pa 
mi conferma che la richiesta è simile a https://github.com/fattureincloud/api/discussions/377?

Grazie
Mauro

**Reply by andry80pa** [reply]
Buongiorno, si confermo

**Reply by valmoz** [reply]
ok, allora chiudo qui in quanto è gestita nell'altra discussion.

Grazie
Mauro
